<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="OneContactFrame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="CheckContactTxt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="SearchConLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Contacts</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="OneContactCheck">
     <property name="geometry">
      <rect>
       <x>-7</x>
       <y>100</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="allContacts">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Check all contacts</string>
    </property>
   </widget>
   <widget class="QLabel" name="SelectionLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Make your selection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SpecificContact">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check specific contact</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="NewContact">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter new contact</string>
    </property>
   </widget>
   <widget class="QFrame" name="AllContactsFrame">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="AllContactsTxt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>231</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="AllContactsLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>All Contacts</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="NewContactFrame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="NameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="NameTxtBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="emailLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="PhoneLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="EmailTxtBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="PhoneTxtBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveContactButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Contact</string>
     </property>
    </widget>
    <widget class="QLabel" name="ErrorLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton_3</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NewContact</sender>
   <signal>clicked()</signal>
   <receiver>NewContactFrame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NewContact</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewContactFrame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allContacts</sender>
   <signal>clicked()</signal>
   <receiver>AllContactsFrame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allContacts</sender>
   <signal>toggled(bool)</signal>
   <receiver>AllContactsFrame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allContacts</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allContacts</sender>
   <signal>toggled(bool)</signal>
   <receiver>AllContactsTxt</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpecificContact</sender>
   <signal>clicked()</signal>
   <receiver>OneContactFrame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpecificContact</sender>
   <signal>toggled(bool)</signal>
   <receiver>OneContactFrame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
